src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/KeyworkerService.java

```diff
@@ -351,6 +355,16 @@ private KeyworkerDto convertToKeyworkerDto(StaffLocationRoleDto dto) {
         return keyworkerDto;
     }
 
+    private KeyworkerDto decorateWithKeyworkerData(KeyworkerDto keyworkerDto) {
+        final Keyworker keyworker = keyworkerRepository.findOne(keyworkerDto.getStaffId());
+        final Integer allocationsCount = repository.countByStaffIdAndAgencyIdAndActive(keyworkerDto.getStaffId(), keyworkerDto.getAgencyId(), true);
+
+        keyworkerDto.setCapacity((keyworker != null && keyworker.getCapacity() != null) ? keyworker.getCapacity() : capacityDefault);
+        keyworkerDto.setStatus(keyworker != null ? keyworker.getStatus() : KeyworkerStatus.ACTIVE);
+        keyworkerDto.setNumberAllocated(allocationsCount);
+        return keyworkerDto;
+    }
+
```

Comment:
need confirmation on whether we are ignoring "hoursWorked" from elite with regards to capacity